readme.md

```diff
@@ -0,0 +1,237 @@
+# meta-xt-prod-devel
+
+## Content
+This document is long enough to has it's own content. So below you will find:
+* What is meta-xt-prod-devel
+* Quick start
+  * Some basic and most used commands
+* Usage and setup
+  * Serial over USB
+  * SSH over ethernet
+  * What to do "on board"
+* How to build
+  * Few build hints
+* How to install
+  * Write to microSD card
+  * Write to eMMC
+* How to prepare board (loaders and u-boot settings)
+* Supported boards
+* References
+
+Important note.
+All PC-related steps described bellow are intended to be run on Linux, and were tested on Ubuntu 18.
+Windows and MacOS are not supported, sorry.
+
+## What is meta-xt-prod-devel
+
+This is meta layer that allows you to run Linux+Linux or Linux+Android at same time under Xen hypervisor on some Renesas' R-CAR boards.
+List of fully or conditionaly supported boards can be found at the end of this document.
+Taking into account that you are interested in these sources, we suppose that you have some understanding what is Xen hypervisor.
+In case you need additional info look into last section of this document - "References".
+What prod-devel is?
+* it is some reference implementation that demonstrate simultanious run of few OS on same hardware
+Why it is under Xen hypervisor?
+* it can reboot promptly in case of some failure met
+* it can share hardware with some other special OS
+
+
+## Quick start
+
+As far as meta-xt-prod-devel is meta layer, it's not intended to be build by itself, but as part of bigger system.
+So, you need to use our tool to build required system.
+```
+git clone git@github.com:xen-troops/build-scripts.git
+cd build-scripts
+python ./build_prod.py --machine h3ulcb-4x2g-kf --product devel \
+    --config local-devel-u.cfg --with-local-conf --with-do-build
+```
+Build takes about four hours on iCore7 with 32GB and SSD.
+You need to have 500GB of free space on disk mentiond in [path] section of local-devel-u.cfg.
+
+Go to `deploy` folder and run
+```
+./mk_sdcard_image.sh -p . -d ./image.img -c devel
```

Comment:
or **-c gen3** to build DomU based system. I noticed down the text the DomU was mentioned many times, so probably we need expand options?

Reply:
Thanks, '-c gen3' is added.
Also I agree, that info regarding other option is needed. Looks like TODO item.